utils/appliance.py

```diff
@@ -39,6 +39,25 @@
     from utils.hosts import setup_providers_hosts_credentials
 
 
+class logger_wrap(object):
```

Comment:
Why would you want kwarg to be modifiable and where would you pass that in at the decorator level?